recipes/wannier90/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "wannier90" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wannier-developers/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 40651a9832eb93dec20a8360dd535262c261c34e13c41b6755fa6915c936b254
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - libblas
+    - liblapack
+  run:
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/wannier90.x
```

Comment:
Can you try executing either of these in any way? or is that too complex?

Reply:
I thought that is fine if I use `make test-serial` - meaning I test the executable after the compilation.

Reply:
It's good to know if the package survives being relocated and that all of the necessary run requirements are in place

Reply:
Though of course, if running a command takes a long time or it requires special input data then it's not necessarily worth the hassle. It's up to you.

Reply:
I can try to add the input test files and then execute the test after the copy. But I would prefer to do this once the recipe is merged. I also want to try building a Version for Mac Os X then.